Sources/MongoKitten/QueryBuilder.swift

```diff
@@ -281,6 +287,15 @@ public indirect enum AQT {
     
     /// Whether nothing needs to be matched. Is always true and just a placeholder
     case nothing
+    
+    /// Whether the String value within the `key` contains this `Value` as a String.
+    case contains(key: String, val: ValueProtocol)
```

Comment:
Good point. I was just sticking with the established pattern that took values. I'll switch it to `String`.